www/webapp/src/views/DomainSetup.vue

```diff
@@ -74,7 +74,7 @@
             <v-col cols="auto">DS Format</v-col>
             <v-spacer></v-spacer>
             <v-col class="text-right">
-              <v-btn @click="copyToClipboard(ds.join('\n'))" text small>
+              <v-btn @click="copyToClipboard(ds.join('\n'))" text small aria-label="Copy DS">
```

Comment:
```suggestion
              <v-btn @click="copyToClipboard(ds.join('\n'))" text small aria-label="Copy DS information">
```

There might be several.